docs/api/access-control.md

```diff
@@ -1,68 +1,37 @@
 <!--[meta]
 section: api
 title: Access Control
+order: 5
 [meta]-->
 
-# Access Control API
+# Access Control
 
 Control who can do what with your GraphQL API.
 
-_Note: This is the API documentation for Access Control. For getting started,
-see the [Access Control Guide](../guides/access-control.md) or the
-[Authentication Guide](../guides/authentication.md)._
+_Note_: This is the API documentation for Access Control. For getting started,
+see the [Access Control Guide](/guides/access-control.md) or the
```

Comment:
These link changes will mean they no longer work on GitHub which we were trying to maintain for folks browsing the source. We have a gatsby plugin which rewrites them to work for the website on build.